src/main/java/duke/exceptions/CommandNotValidException.java

```diff
@@ -0,0 +1,18 @@
+package duke.exceptions;
+
+/**
+ * Error class for invalid commands.
+ */
+public class CommandNotValidException extends DukeException {
+    /**
+     * Constructs a CommandNotValidException with specific message for invalid commands.
+     */
+    public CommandNotValidException() {
+        super("Command not valid. Please use \"todo\", \"deadline\""
+                + "or \"event\" followed by task description to add new tasks. "
```

Comment:
I like the verbosity here, although I wonder if it could be made more user-friendly?